.github/ISSUE_TEMPLATE/Bug_report.md

```diff
@@ -20,32 +20,17 @@ A clear and concise description of what you expected to happen.
 **Screenshots**
 If applicable, add screenshots to help explain your problem.
 
+**Server details:**
+- Nextcloud version: [e.g. 26.0.1]
+- PHP Version: [e.g. 8.2]
+- Database: [e.g. MariaDB 10.6]
+
 **Client details:**
- - OS: [e.g. iOS]
- - Browser: [e.g. chrome, safari]
- - Version: [e.g. 22]
+ - OS: [e.g. Windows/macOS/Ubuntu]
+ - Browser: [e.g. Firefox, Chrome]
+ - Browser version: [e.g. 22]
  - Device: [e.g. iPhone6, desktop]

```

Comment:
wouldn't something like `desktop, mobile, tablet` represent the info better? it doesn't really matter what exact device it is, just it's class

Reply:
The exact device might be relevant for mobile issues where for example on Android the on screen keyboard might introduce different behavior based on the vendors to the contenteditable HTML attribute

Reply:
I see